concordia/static/scss/action-app.scss

```diff
@@ -186,23 +187,15 @@ main {
     }
 
     .asset-tooltip {
-        width: calc(100% + ($asset-thumbnail-gap * 2));
-        height: calc(100% + ($asset-thumbnail-gap * 2));
-
-        margin: $asset-thumbnail-gap * -1;
         padding: 14px;
```

Comment:
Should this value also change? On the small size it means there's a pretty large border (~20% of the total width) around the text which looks a bit wasteful.

Reply:
I'll leave this to @jbresner

Reply:
That's my fault. I never mocked that up and only had the warning label displayed ("Another person is working on this asset") on the very small size displayed in the asset list. However, I think the padding for the warning label on the small size is closer to what @acdha was suggesting for the asset title. @klee72 offered to go ahead and make the change.